resources/devices/mobile-phone/htc.json

```diff
@@ -1949,19 +1936,6 @@
             "isTablet": "false"
         }
     },
-    "HTC Touch HD2 T8585": {
```

Comment:
There already was an "HTC T8585" so I just modified the patterns to use that device if they were using this one.